Dockerfile.build

```diff
@@ -0,0 +1,3 @@
+FROM maven:3-jdk-7-onbuild
+
+CMD ["mvn test"]
```

Comment:
Let's remove this for now. This would be useful to get the s3-tests to run locally so I opened #146.